<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="priceNameEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="priceCodespinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="priceDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Price List</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="orderNameEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>List Of Orders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addingPriceButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addingOrderButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="priceCorrectButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orderCorrectButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="codeSpinBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>410</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantitySpinBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>410</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Код:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>410</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="priceTable">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Код</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена за 1 шт.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="orderTable">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>511</width>
      <height>151</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Номер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Код</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Заказчик</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addGoodsButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>410</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showPriceButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="priceHideButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="priceChoiseSpinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="priceDeleteButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>80</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Заказчик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orderDeleteButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="orderChoiseSpinBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showOrderButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orderHideButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>391</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Накладная</string>
    </property>
   </widget>
   <widget class="QPushButton" name="formingButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Сформировать накладную</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер заказа:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Общая стоимость:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="formingTable">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Код товара</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="numberLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="costLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>107</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Код:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Цена:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Название:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="priceFileAddingButton">
    <property name="geometry">
     <rect>
      <x>357</x>
      <y>24</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить из файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orderFileAddingButton">
    <property name="geometry">
     <rect>
      <x>987</x>
      <y>20</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить из файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="priceFileWritingButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Записать в файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orderFileWriteButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Записать в файл</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Сортировать по коду</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сортировать по цене</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
